.github/workflows/integration-tests.yml

```diff
@@ -102,24 +102,89 @@ jobs:
 
   psycopg2-check:
     name: "Test psycopg2 build version"
-    runs-on: ${{ matrix.scenario.platform }}
+    runs-on: ${{ matrix.platform }}
     strategy:
       fail-fast: false
       matrix:
-        scenario:
-          - {platform: ubuntu-latest, psycopg2-name: psycopg2}
-          - {platform: macos-12, psycopg2-name: psycopg2-binary}
+        platform: [ubuntu-22.04, macos-12]
```

Comment:
Just making sure, is the right macos? I know we have trouble with that before.

Reply:
Yeah, that used to be the equivalent of `macos-latest`. Now it's `macos-14`. The difference is that `macos-14` does not come with `py38` installed. However, since we're installing `py38` as a step here, it may not matter. `macos-12` is just playing it safe.